src/pykeen/losses.py

```diff
@@ -1280,6 +1281,142 @@ def forward(
         return loss
 
 
+@parse_docdata
+class InfoNCELoss(SetwiseLoss):
+    r"""An implementation of the InfoNCE loss with additive margin proposed by [wang2022]_.
```

Comment:
double check this is the right wang2022 reference and it's already included in the docs/references.rst

Reply:
63acfd1